readme.md

```diff
@@ -20,12 +20,15 @@ An exercise in iterative development by [Matt Stauffer](http://mattstauffer.co/)
 1. Clone the repository locally
 2. Install dependencies with `composer install`
 3. Copy `.env.example` to `.env` and modify its contents to reflect your local environment
-4. Configure a web server, such as the built-in PHP web server, to serve this site using the +public+ directory as its root
+5. Go to https://github.com/settings/developers and create a Oauth app 
```

Comment:
Looks like you have an accidental duplication here, of 5 + 6 + 5? Can you clean that up? Thanks!

Reply:
Sorry about that, had some internet connection issues at the train :)
Thanks for the review - It should be all good now.

Reply:
Looks like you've now deleted the previous bullet point 4 :)

Reply:
That's so weird, next time I should not use Github's inline editor ;) should be all okay now